### PR TITLE
Add n-best_with_tree option

### PR DESCRIPTION
**Pull request by MorinoseiMorizo**

### CODE REVIEW

File: src/lib/travatar-runner.cc [hunk 1]
```diff
@@ -76,11 +76,21 @@ void TravatarRunnerTask::Run() {
     if(nbest_collector_ != NULL) {
         ostringstream nbest_out;
         BOOST_FOREACH(const boost::shared_ptr<HyperPath> & path, nbest_list) {
-            nbest_out
-                << sent_
-                << " ||| " << Dict::PrintWords(path->GetTrgData())
-                << " ||| " << path->GetScore()
-                << " ||| " << Dict::PrintSparseVector(path->CalcFeatures()) << endl;
+            if(runner_->GetNbestWithTree() == false){
```
Comment (neubig) [inline src/lib/travatar-runner.cc]: The "if" statement here doesn't need to be around all 5 lines. Could you put it just around the line "<< " ||| " << path->GetTreeStr()"

**Comment by neubig** [comment]
And for the option name, either is OK with me.

**Comment by MorinoseiMorizo** [comment]
Thank you for the comments.
I changed the name to "nbest_tree", and fixed if statement.

**Comment by neubig** [comment]
Looks good to me. Thanks!

**Pull request merged**